proj/src/main/webapp/script.js

```diff
@@ -40,9 +40,23 @@ function addUserToTag(reference){
 // Creates a new chat given a tag and adds the current user as a member
 function createNewChatWithUser(tag){
     console.log("creating new chat with tag: " + tag);
+    var time = new Date().getTime();
+    var messageContent = "Welcome to the " + tag + " chat!";
     var newChat = {
-        "name" : tag,
-        "tag" : tag,
+        "chatInfo" : {
+            "name" : tag,
+            "tag" : tag,
+            "lastMessage" : messageContent,
+            "timestamp" : time
```

Comment:
everything looks good to me, but i'm just realizing we'll probably need the user id here too... sorry 🤒